client/src/components/menubar/style.ts

```diff
@@ -0,0 +1,75 @@
+import styled from "@emotion/styled";
+import { spacesS } from "../theme";
+import { getFeatureFlag } from "../../util/featureFlags/featureFlags";
+import { FEATURES } from "../../util/featureFlags/features";
+
+export const fullVerticalScreenWidth = 500;
```

Comment:
nit: Since this is a constant, let's use ALL_CAPS and suffix with unit for readability 😄 🙏 

Discussion: I removed `FULL` from the variable name, since it doesn't seem like there's a less than full vertical state? Or maybe I missed it 😆 PTAL thank you!

```suggestion
export const VERTICAL_THRESHOLD_WIDTH_PX = 500;
```

Reply:
There are two different breakpoints - I changed them to FIRST_VERTICAL_THRESHOLD_WIDTH_PX & MAX_VERTICAL_THRESHOLD_WIDTH_PX 👍